jbpm-xes/pom.xml

```diff
@@ -127,10 +137,57 @@
       <groupId>org.postgresql</groupId>
       <artifactId>postgresql</artifactId>
     </dependency>
+    
+    <!-- needed for jdk 11 -->
+    <dependency>
+      <groupId>org.jboss.spec.javax.xml.bind</groupId>
+      <artifactId>jboss-jaxb-api_2.3_spec</artifactId>
+    </dependency>
+
+    <dependency>
+      <groupId>com.sun.xml.bind</groupId>
+      <artifactId>jaxb-impl</artifactId>
+    </dependency>
+    
+    <dependency>
+      <groupId>com.sun.xml.bind</groupId>
+      <artifactId>jaxb-core</artifactId>
+    </dependency>
+    
+    <dependency>
+      <groupId>javax.activation</groupId>
+      <artifactId>activation</artifactId>
+    </dependency>
 
   </dependencies>
 
   <build>
+   <pluginManagement>
+     <plugins>
+       <plugin>
+          <groupId>org.apache.maven.plugins</groupId>
+          <artifactId>maven-enforcer-plugin</artifactId>
+          <executions>
+            <execution>
+              <id>ban-blacklisted-dependencies</id>
+              <goals>
+                <goal>enforce</goal>
+              </goals>
+              <configuration>
+                <rules>
+                  <bannedDependencies>
+                    <includes>
+                      <include>commons-logging:*</include>
+                    </includes>
+                  </bannedDependencies>
```

Comment:
Why was this banned?

Reply:
It's banned at kie-parent pom:
 <!-- Ban blacklisted logging deps (we only use SLF4J + Logback) -->
         <!-- In case of transitive dependency, exclude it and use 'org.slf4j:jcl-over-slf4j' instead -->
                      <exclude>commons-logging:commons-log*</exclude>

As it's needed for working (BasicDataSource depends on org.apache.commons.logging.LogFactory), Cristiano and me agreed to override the enforcer plugin at pluginManagement for skipping this banned dependency.